frontend/app/models/user-store/user-store.ts

```diff
@@ -20,13 +20,13 @@ const toastAndroid = (message: string) => {
   ToastAndroid.showWithGravity(
     message,
     ToastAndroid.LONG,
-    ToastAndroid.TOP,
+    ToastAndroid.CENTER,
```

Comment:
The only concern I have with the toastr in the center is if they make the tests fail, like how the warning made the tests not able to press the buttons.

Have you tried that?

Reply:
oh true, ill change it back to top